customer-satisfaction/run_pipeline.py

```diff
@@ -7,20 +7,17 @@
 from zenml.integrations.mlflow.mlflow_utils import get_tracking_uri

```

Comment:
Would it make sense to run from a YAML file instead using the new method described here: https://docs.zenml.io/advanced-guide/pipelines/settings#method-3-configuring-with-yaml